.github/workflows/test.yml

```diff
@@ -0,0 +1,21 @@
+name: Test spot_ros2
```

Comment:
Please rename this "spot_ros2 CI". Normally it doesn't matter but in this case, I believe this string shows up in the Readme as the CI badge (so fairly obvious and should have a clear name).

Reply:
Done.